Dart/testSrc/com/jetbrains/lang/dart/completion/reference/DartReferenceCompletionInLibraryRootTest.java

```diff
@@ -54,6 +54,10 @@ public void testMixin1() throws Throwable {
     doTest();
   }
 
+  public void testNonNullRef() throws Throwable {
```

Comment:
Next time when you'll need to write test for completion take a look at DartCompletionTest. It looks more powerful and easier to write and maintain.